cluster/images/hyperkube/Makefile

```diff
@@ -1,6 +1,9 @@
-# build the hyperkube image.
+# Build the hyperkube image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-VERSION=v1.1.2
+REGISTRY?="gcr.io/google_containers"
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/hack/lib/version.sh#L33

KUBE_GIT_VERSION is the semantic version we use elsewhere.